### ISSUE TITLE
Edit Base Schedule Index attribute

### ISSUE THREAD

**Issue by subcritical71** [jeroenterheerdt/HAsmartirrigation#67]
I am missing something. During installation of your project I didn't set the correct values for my zones. I've been trying to edit the attribute but it just doesn't seem to stick after a reboot of Home Assistant. Could you tell me the proper location to edit 'Base Schedule Index'? In particular I want to modify;

1. Number of sprinklers
2. Flow
3. Throughput
4. Reference evapotranspiration
5. Peak evapotranspiration
6. Area
7. Precipitation rate
8. Base schedule index minutes

Do any of these self recalculate based on the value of the other items? For example, item 1) and 2) give 3). Is 3 automatically calculated?

Thanks in advance.

**Comment by jeroenterheerdt** [comment]
FAQ: If you want to go back and change your settings afterwards, you can either delete the instance and re-create it or edit the entity registry under config/.storage at your own risk.

**Comment by jeroenterheerdt** [comment]
You need to provide 1, 2, 3, 4 and 6. 5/7/8 are calculated so you cannot configure them. The UI config you went through the first time automatically calculates 3 for you.

**Comment by subcritical71** [comment]
Thank you for the nudge. While editing the core.config_entries file I noticed that the following were listed under options;

"options": {
                    "lead_time": 0,
                    "maximum_duration": 1800,
                    "force_mode_duration": 600,
                    "show_units": false,
                    "auto_refresh": true,
                    "auto_refresh_time": "23:00",
                    "initial_update_delay": 300,
                    "coastal": false,
                    "estimate_solrad_from_temp": true
                },

All of the items are configurable under Configuration > Integrations > Smart Irrigation Card - Configure (for each zone). Is there a reason why the data and name fields couldn't be included in the configure command? May be an opportunity for improvement if there is not a reason to not do this.

Thanks for the plugin. I see a lot of promise in it once I completely figure it out. I just found HA so I'm on the bottom of the learning curve at the moment. My plan is to put a guide together to help others that go down this route (and have a way for me to redo things if necessary!).
